pulley/src/decode.rs

```diff
@@ -655,3 +682,43 @@ macro_rules! define_extended_decoder {
     };
 }
 for_each_extended_op!(define_extended_decoder);
+
+fn unwrap_uninhabited<T>(res: Result<T, Uninhabited>) -> T {
+    match res {
+        Ok(ok) => ok,
+    }
+}
+
+#[allow(missing_docs)]
+pub mod operands {
```

Comment:
Maybe some docs for the module, even if we are skipping them for the individual functions?